pkg/plugins/pluginrpc-gen/README.md

```diff
@@ -30,7 +30,7 @@ Where:
 - `--type` is the name of the interface to use
 - `--name` is the subsystem that the plugin "Implements"
 - `-i` is the input file containing the interface definition
-- `-o` is the output file where the the generated code should go
```

Comment:
Actually when I use a grep to match string `the the ` in the whole repo, I found that there are some more cases with this deuplicated `the`. Could you help us to fix those for us in this PR?